tests/EdFi.Ods.Api.IntegrationTests/StudentUSIToEducationOrganizationIdAuthViewTests.cs

```diff
@@ -53,12 +53,12 @@ public void When_enroll_student_in_school_should_not_access_from_other_school()
             Builder
                 .AddStudentSchoolAssociation(9702, studentUSI, Builder.TestGradeLevelDescriptorId)
                 .Execute();
-
-            AuthorizationViewHelper.ShouldNotContainStudentInOtherSchoolOrDistrict(Connection, "StudentUSIToEducationOrganizationId", (9702, studentUSI));
+            var expectedTuples = new[] { (255901001, studentUSI), (255901044, studentUSI), (255901107, studentUSI) };
+            AuthorizationViewHelper.ShouldNotContainTuples(Connection, "StudentUSIToEducationOrganizationId", expectedTuples);
```

Comment:
I think we should set up a second school here that they are not associated with and then we'd expect it not to contain that tuple. We want to make sure the schools we're checking actually exist and not rely on the populated template to provide that information.

Reply:
done